src/utils/parsingService.js

```diff
@@ -1,25 +1,20 @@
-'use strict'
-
 const _ = require('lodash')
 const Service = require('trails-service')
 
-/**
- * @module ParsingService
- */
-module.exports = class ParsingService extends Service {
+export default class ParsingService {
 
   /**
    * Parse object with selection info and return install command
    */
-  parseSelections (settings) {
+  static parseSelections (settings) {
```

Comment:
No need to make the ParsingService a class if all the methods are going to be static.